modules/navier_stokes/test/tests/finite_volume/ins/mms/skew-correction/skewed-vortex.i

```diff
@@ -1,35 +1,32 @@
-mu=1.1
-rho=1.1
+mu=1.0
+rho=1.0
```

Comment:
Why this change? I often do these settings just to make sure that I didn't actually forget a rho or mu coefficient somewhere.

Reply:
I set those to 1.0 because the formulation in the paper I used didn't bother adding the material properties in their formulation. The analytic solutions are only valid for this setting.

Reply:
got ya